### PR TITLE
Remove support for blowfish

### PR DESCRIPTION
**Pull request by bigs**
Removes a reference to blowfish

### CODE REVIEW

File: crypto/key.go [hunk 1]
```diff
@@ -191,10 +191,6 @@ func KeyStretcher(cipherType string, hashType string, secret []byte) (StretchedK
 	case "AES-256":
 		ivSize = 16
 		cipherKeySize = 32
-	case "Blowfish":
```
Comment (Stebalien) [inline crypto/key.go]: We should have a default case, right?

Comment (Kubuxu) [reply crypto/key.go]: We for sure need a default case here.

Comment (bigs) [reply crypto/key.go]: aye, can do

**Comment by bigs** [comment]
bump @kubuxu

**Pull request merged**